docs/features.md

```diff
@@ -0,0 +1,47 @@
+## Shop-easy platform features
+
+There are two user roles in our platform. They are
+    - Admin User
+    - Student User
+
+### Admin User:
+
+Admin has administrative previliges and responsible for pantry management such as Inventory management and order tracking etc.,
+
+### Student User:
+
+Student can access the pantry platform for convinent and user friendly purchases of the listed items.
+
+### Admin features:
+
+- **Admin authentication**: Admin login worflow with username and password.
+- **Inventory Management**: Manage inventory to add, update and delete items based on the category.
+- **Order Tracking**: Tracking student orders, their status and history.
+- **User Management**: Administrative previliges to add, update and delete students.
+
+### Student features:
```

Comment:
##Student Features:
Student Authentication: Since the student is a specific college student where the pantry is operating, can we consider that all these students are registered by default by skipping the registration part and we can just do login authentication?
**Favorites List:**Students should be able to Mark favorites and View the Marked Favorites and Remove Favorites.
Notifications: Students should be able to view notifications on notifications tab if any of their favorited items are restocked.

Reply:
Addressed